### PR TITLE
Add GitVersion.yml actual binary will come later when it is decided w…

### PR DESCRIPTION
**Pull request by bbowman**
…hich version to use and how to consume it.

### CODE REVIEW

File: GitVersion.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    is-release-branch: true
+    is-mainline: true
+  develop:
+    regex: develop
+    increment: None
+    tag: 'dev'
+    track-merge-target: true
+    tracks-release-branches: true
+    is-release-branch: false
+  .*:
```
Comment (memontic-ms) [inline GitVersion.yml]: Is increment an optional config? Briefly looking at the docs it seems this should be "increment: Inherit".

**Comment by memontic-ms** [comment]
:shipit:

File: GitVersion.yml (hunk 1 above)
Comment (bbowman) [inline GitVersion.yml]: I can try that out.

Comment (bbowman) [inline GitVersion.yml]: or actually none. I think.

**Pull request merged**